src/reactapp/src/api/paypalExpress/createCustomerToken/mutation.js

```diff
@@ -0,0 +1,25 @@
+export const CREATE_PAYPAL_EXPRESS_TOKEN = `
+mutation createPaypalExpressToken(
+    $cartId: String!
+    $returnUrl: String!
+    $cancelUrl: String!
+  ) {
+    createPaypalExpressToken(
+      input: {
+        cart_id: $cartId
+        code: "paypal_express"
```

Comment:
I know the payment method code is not going to change. Still, we should avoid hard coding `paypal_express` here. We should pass it in `variable` here. This way we can make sure our mutation works even if the implementation changes the value in future.

Reply:
Done